pycbc/events/cuts.py

```diff
@@ -182,8 +226,10 @@ def apply_trigger_cuts(triggers, trigger_cut_dict):
             value = get_chisq_from_file_choice(triggers, chisq_choice)
             # Apply any previous cuts to the value for comparison
             value = value[idx_out]
-        elif parameter in triggers.file[triggers.ifo]:
-            # parameter can be read direct from the trigger file
+        elif (parameter in triggers
+                or (hasattr(triggers, "file")
+                    and parameter in triggers.file[triggers.ifo])):
+            # parameter can be read direct from the trigger dictionary / file
```

Comment:
This change is to make things easier for the testing - we can use a dictionary rather than a ReadByTemplate object